### PR TITLE
rename low bit type name

### PR DESCRIPTION
**Pull request by qiuxin2012**
## Description

rename qx_0 to sym_intx, change qx_1 to asym_intx

### 1. Why the change?

low bit type qx_0 and qx_1 are not good.

### 2. User API changes

rename qx_0 to sym_intx, change qx_1 to asym_intx

### 3. Summary of the change 

rename q4_0 to sym_int4
rename q4_1 to asym_int4
rename q5_0 to sym_int5
rename q5_1 to asym_int5
rename q8_0 to sym_int8

### 4. How to test?
- [ ] N/A
- [ ] Unit test
- [ ] Application test
- [ ] Document test
- [ ] ...

### CODE REVIEW

File: python/llm/src/bigdl/llm/transformers/model.py [hunk 1]
```diff
@@ -95,8 +95,8 @@ def from_pretrained(cls,
     def convert_quant(cls, model, q_k, *args, **kwargs):
         from .convert import ggml_convert_quant
         invalidInputError(q_k in ggml_tensor_qtype,
-                          f"Unknown load_in_low_bit value: {q_k},"
-                          f" excepted q4_0, q4_1, q5_0, q5_1, q8_0.")
+                          f"Unknown load_in_low_bit value: {q_k}, excepted"
```
Comment (leonardozcm) [inline python/llm/src/bigdl/llm/transformers/model.py:98]: expect

**Review by leonardozcm — APPROVED** [review]
LGTM

File: python/llm/src/bigdl/llm/transformers/model.py [hunk 2]
```diff
@@ -49,8 +49,8 @@ def from_pretrained(cls,
 
         if bigdl_transformers_low_bit:
             invalidInputError(bigdl_transformers_low_bit in ggml_tensor_qtype,
-                              f"Unknown load_in_low_bit value: {bigdl_transformers_low_bit},"
-                              f" excepted q4_0, q4_1, q5_0, q5_1, q8_0.")
+                              f"Unknown bigdl_transformers_low_bit value: {bigdl_transformers_low_bit},"
```
Comment (jason-dai) [inline python/llm/src/bigdl/llm/transformers/model.py:52]: add python doc for the API

File: python/llm/src/bigdl/llm/transformers/model.py [hunk 3]
```diff
@@ -29,6 +29,17 @@ class _BaseAutoModelClass:
     def from_pretrained(cls,
                         *args,
                         **kwargs):
+        """
+        Load a model from a directory or the HF Hub. Use load_in_4bit or load_in_low_bit parameter
+        the weight of model's linears can be loaded to low-bit format, like int4, int5 and int8.
+
+        Two new arguments are added rather than huggingface's from_pretrained, listed below.
```
Comment (jason-dai) [inline python/llm/src/bigdl/llm/transformers/model.py:36]: Two new arguments are added to extend Hugging Face's from_pretrained method as follows:

**Pull request merged**